app/boards/shields/tester_pro_micro/tester_pro_micro.keymap

```diff
@@ -0,0 +1,63 @@
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+
+#define PIN_MACRO(name, pin) \
+/ { \
+    macros { \
+        name: name { \
+            compatible = "zmk,behavior-macro"; \
+            wait-ms = <5>; \
+            tap-ms = <5>; \
+            #binding-cells = <0>; \
+            bindings = <&kp P &kp I &kp N &kp SPACE>, pin, <&kp ENTER>; \
+        }; \
+    }; \
+};
+
+PIN_MACRO(pin0, <&kp N0>)
+PIN_MACRO(pin1, <&kp N1>)
+PIN_MACRO(pin2, <&kp N2>)
+PIN_MACRO(pin3, <&kp N3>)
+PIN_MACRO(pin4, <&kp N4>)
+PIN_MACRO(pin5, <&kp N5>)
+PIN_MACRO(pin6, <&kp N6>)
+PIN_MACRO(pin7, <&kp N7>)
+PIN_MACRO(pin8, <&kp N8>)
+PIN_MACRO(pin9, <&kp N9>)
+PIN_MACRO(pin10, <&kp N1 &kp N0>)
+PIN_MACRO(pin14, <&kp N1 &kp N4>)
+PIN_MACRO(pin15, <&kp N1 &kp N5>)
+PIN_MACRO(pin16, <&kp N1 &kp N6>)
+PIN_MACRO(pin18, <&kp N1 &kp N8>)
+PIN_MACRO(pin19, <&kp N1 &kp N9>)
+PIN_MACRO(pin20, <&kp N2 &kp N0>)
+PIN_MACRO(pin21, <&kp N2 &kp N1>)
+
+#define PM(id) &pin##id
```

Comment:
Nit, but this seems a bit pointless to me given what it expands to isn't very verbose either.

Reply:
Good point, this is an artifact left over from when I was trying to get "message if shorted on boot" working. Removed.